CSharp/BatchExplorer/Models/ModelBase.cs

```diff
@@ -106,13 +106,17 @@ private List<PropertyModel> ObjectToPropertyModelRecursive(object o, List<string
                         //Don't track null properties in order to keep the list short
                         //propertyModel = new SimplePropertyModel(propInfo.Name, string.Empty);
                     }
+                    else if ((propertyType == typeof(TimeSpan) || propertyType == typeof(TimeSpan?)) && propertyValue.Equals(TimeSpan.MaxValue))
+                    {
+                        propertyModel = new SimplePropertyModel(propInfo.Name, "Unlimited");
```

Comment:
@paselem I don't have strong feelings but to me something like "MaxWallClockTime: Unlimited" reads more naturally than "MaxWallClockTime: TimeSpan.MaxValue".  In addition, this is a generally useful tool that may be used by non-.NET devs to view the state of jobs and pools instead of writing custom code or using the CLI (they don't need to understand the C# code to run it!).  But if people prefer text other than "Unlimited" then I can change it -- I just found the "109000 days" format confusing.